src/components/HistoryModal/Content.tsx

```diff
@@ -19,7 +19,7 @@ import { faExternalLinkAlt, faUndo } from '@fortawesome/free-solid-svg-icons';
 import { useToggle } from '../../hooks/useToggle';
 import { RefundConfirmationModal } from '../RefundConfirmationModal/RefundConfirmationModal';
 import { truncate } from '../../utils/string';
-import { capitalize } from 'lodash';
+import capitalize from 'lodash/capitalize';
```

Comment:
As I remember in this situation lodash bundles up completely despite the relative import and tree shaking didn't work. lodash-es packages do the trick.

Reply:
![image](https://user-images.githubusercontent.com/26635501/128917094-d657296c-1eb0-4f08-8a2f-607d93027129.png)
nope, all is working

Reply:
Cool. What plugin do you use for that?

Reply:
import cost